xml/windows-appendix.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<appendix xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="windows-appendix">
+ <title>Appendicies</title>
```

Comment:
```suggestion
 <title>Appendices</title>
```

Reply:
Acc. to style guide, actually "appendixes", https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-terminology (though I like the version with c better too)

Reply:
I'll roll with that.